BEIMA.DB.Schemas/building-schema

```diff
@@ -0,0 +1,6 @@
+{
+    "buildingId": String,
```

Comment:
Looking at the email, it looked like he also wanted to have the building number.

Reply:
The buildingId field is a randomly generated UID to allow the ability to modify properties of the building without having to modify all documents in the database. The building number (or name) will be stored in the "name" field. Any other information can be stored in the "description" field.

Reply:
I think that he wants number and name to be two separate entities though right? In the parameter column he lists it as "Building # & name"

Reply:
We should change the 'name' field to be 'number' and 'description' to 'name' to make that connection a bit clearer.

Reply:
@josephmoore99 ah I see, yes the naming of the fields was definitely a bit confusing. I will use @AshlynAdamson's suggestion and change it to "name" and "number".

Reply:
This has been fixed